package.json

```diff
@@ -6,11 +6,14 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.3.3",
     "@testing-library/user-event": "^7.1.2",
+    "algolia-places-react": "^1.5.1",
     "axios": "^0.19.0",
+    "dotenv": "^8.2.0",
```

Comment:
what exactly do we need dotenv for?

Reply:
For the algolia app id and secrets. Setting them to the .env file

Reply:
but we don't need dotenv, simply use REACT_APP_APP_ID and REACT_APP_APP_KEY